mariadb-10.11.advisories.yaml

```diff
@@ -4,12 +4,12 @@ package:
   name: mariadb-10.11
 
 advisories:
-  - id: CGA-69ff-4xff-qv2w
```

Comment:
I'm not sure why these got updated...

Reply:
It's just a reordering. It looks weird. This is a known vestigial thing that affects some advisories files. There's a command to see what's actually being changed, e.g. I just ran it for your branch:

```console
$ wolfictl adv diff
Auto-detected distro: Wolfi

( - removed / ~ modified / + added )

~ document "mariadb-10.11"
  ~ advisory "CGA-x53v-hg6v-7rvv"
    + event "false-positive-determination" @ 2024-06-25T18:11:51Z
~ document "mariadb-11.2"
  ~ advisory "CGA-5cmg-w6mr-wx73"
    + event "false-positive-determination" @ 2024-06-25T18:13:23Z
```